kubeflow-roles/README.md

```diff
@@ -0,0 +1,74 @@
+# Default Kubeflow ClusterRoles
+
+This manifest package contains the default ClusterRoles Kubeflow uses for defining roles for Kubeflow user Profiles.
+These roles are currently assigned to users by Profiles (profile-controller and kfam) Service with the help of Manage Users page in Central Dashboard.
+
+## Aggregating default Kubernetes ClusterRoles to Kubeflow ClusterRoles
+
+Using [ClusterRole Aggregation](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles) `kfctl` currently labels each of [the default Kubernetes ClusterRoles](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#default-roles-and-role-bindings) to its Kubeflow Roles defined here.
```

Comment:
Should this be updated to reflect the latest decision:
that we aren't labeling the default K8s ClusterRoles, but defining our own.

Reply:
Sure I'll update it.